code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -113,14 +113,143 @@
+	attack_verb = list("punched", "pummeled", "burnt", "seared")
+			
+/obj/item/nullrod/godhand/attack(mob/living/carbon/M, mob/user)
+	if(!isliving(M))
+		return
+	M.IgniteMob()
+	if(!ismob(M))
+		return
+
+	if(istype(M.wear_mask, /obj/item/clothing/mask/cigarette) && user.zone_selected == "mouth")
+		var/obj/item/clothing/mask/cigarette/cig = M.wear_mask
+		if(M == user)
+			cig.attackby(src, user)
+		else
+			if(istype(src, /obj/item/nullrod/godhand))
+				cig.light("<span class='rose'>[user] extends [user.p_their()] hand out to [M], and [cig] ignites.</span>")
+			else
+				cig.light("<span class='notice'>[user] reaches out to [M] with [user.p_their()] hand, and [cig] suddenly ignites.</span>")
+			playsound(src, 'sound/items/lighter/light.ogg', 25, TRUE)
+			M.update_inv_wear_mask()
+	else
+		..()
+		
+/obj/item/nullrod/godhand/Initialize() // Part one of janky code to make Godhand glow
+	. = ..()
+	update_brightness()
+	
+/obj/item/nullrod/godhand/proc/update_brightness() // Part two of janky code to make Godhand glow
+	set_light(2)
+		
+// Light Candles
+/obj/item/candle/attackby(obj/item/R, mob/user)
+	if(istype(R, /obj/item/nullrod/godhand))
+		light("<span class='notice'>[user] holds their hand over the wick of [src] and it starts burning.</span>")
+		return
+	else
+		..()
+
+// Light cigarettes
+/obj/item/clothing/mask/cigarette/attackby(obj/item/R, mob/user)
+	if(istype(R, /obj/item/nullrod/godhand))
+		light("<span class='notice'>[user] calmly raises [user.p_their()] hand to [user.p_their()] [name] and it ignites.</span>")
+		return
+	else
+		..()
+
+// Light pipes with your finger
+/obj/item/clothing/mask/cigarette/pipe/attackby(obj/item/R, mob/user)
+	if(istype(R, /obj/item/nullrod/godhand))
+		light("<span class='notice'>[user] sticks a finger into [src], and upon taking it out, smoke follows.</span>")
+		return
+	else
+		..()
+
+// Burn paper
+/obj/item/paper/attackby(obj/item/R, mob/user)
+	if(istype(R, /obj/item/nullrod/godhand))
+		user.unEquip(src)
+		user.visible_message("<span class='danger'>[user] touches [src] with a finger and it bursts into flames!</span>", "<span class='danger'>You ignite [src] with your finger!</span>")
+		fire_act()
+		return
+	else
+		..()
+
+// Burn paper airplanes
+/obj/item/paperplane/attackby(obj/item/R, mob/user)
+	if(istype(R, /obj/item/nullrod/godhand))
+		if(!in_range(user, src))
+			return
+		user.unEquip(src)
+		user.visible_message("<span class='danger'>[user] touches [src] with a finger and it bursts into flames!</span>", "<span class='danger'>You ignite [src] with your finger!</span>")
+		fire_act()
+		return
+	else
+		..()
+
+// Burn tickets
+/obj/item/ticket_machine_ticket/attackby(obj/item/R, mob/user)
+	if(istype(R, /obj/item/nullrod/godhand))
+		user.visible_message("<span class='danger'>[user] pokes [src] with [user.p_their()] finger and it bursts into flames!</span>", "<span class='danger'>You poke [src] with your finger and set it on fire!</span>")
+		fire_act()
+		return
```

Comment:
I think you get the gist of it, all of these need to be moved into their original attackby() procs.